Sources/Orbit/Support/Views/SecureTextField.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+import UIKit
+
+struct SecureTextFieldStyle {
+    let textContentType: UITextContentType?
+    let autocapitalization: UITextAutocapitalizationType
```

Comment:
`autocapitalization` does not seem to be respected in secure version. (We can remove it if it is hard to implement)

Reply:
I was thinking does it make sense at all in password context. Let's remove it for now

Reply:
Sure, but the InputField still has this argument and it will be silently ignored 🤷  
We can fix it later, but let's keep FIXME: comment somewhere.